### PR TITLE
fixed tests

### PR DESCRIPTION
**Pull request by TetianaParanich**
Thunderjet tests are fixed

### CODE REVIEW

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js [hunk 1]
```diff
@@ -58,13 +58,13 @@ describe('orders: Receive piece from Order', () => {
       });
   });
 
-  it('C735 Receiving pieces from an order for physical material that is set to create Items in inventory (thunderjet)', { tags: [testType.smoke] }, () => {
+  it('C3506 Catalog a new title which has been ordered and received in Orders (procopovich)', { tags: [testType.smoke] }, () => {
```
Comment (penfriend) [inline cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js:61]: prokopovych

File: cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js [hunk 2]
```diff
@@ -58,13 +58,13 @@ describe('orders: Receive piece from Order', () => {
...
     Orders.searchByParameter('PO number', orderNumber);
-    Helper.selectFromResultsList();
+    Orders.selectFromResultsList(orderNumber);
     Orders.openOrder();
     InteractorsTools.checkCalloutMessage(`The Purchase order - ${orderNumber} has been successfully opened`);
     Orders.receiveOrderViaActions();
-    Helper.selectFromResultsList();
+    Receiving.selectFromResultsList(instanceTitle);
     Receiving.receivePiece(0, caption, barcode);
     Receiving.checkReceivedPiece(0, caption, barcode);
```
Comment (penfriend) [inline cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js:69]: the test fails: did not find accordion with label including "Holdings: Main Library >", did you mean one of:

File: cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js (hunk 1 above)
Comment (penfriend) [inline cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js:61]: prokopovych

Comment (penfriend) [reply cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js:61]: please add in devTeams prokopovych tag

File: cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js:68]: I fixed it

File: cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js]: Done

Comment (TetianaParanich) [reply cypress/e2e/inventory/catalogue-new-ordered-recieved-title.cy.js]: Done

File: cypress/e2e/orders/close-order.cy.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ describe('orders: Close Order', () => {
       .then(orderNumber => {
```
[the remark this answers is not in the document]
Comment (TetianaParanich) [reply cypress/e2e/orders/close-order.cy.js:39]: Done

**Review by penfriend — APPROVED** [review]

File: cypress/e2e/orders/receive-piece-from-order.cy.js [hunk 4]
```diff
@@ -48,7 +48,7 @@ describe('orders: Receive piece from Order', () => {
       .then(orderNumber => {
         cy.visit(TopMenu.ordersPath);
         Orders.searchByParameter('PO number', orderNumber);
-        Helper.selectFromResultsList();
+        Orders.selectFromResultsList(orderNumber);
         Orders.openOrder();
         InteractorsTools.checkCalloutMessage(`The Purchase order - ${orderNumber} has been successfully opened`);
         Orders.receiveOrderViaActions();
```
Comment (penfriend) [inline cypress/e2e/orders/receive-piece-from-order.cy.js:54]: this tests fails: did not find accordion "Holdings: Main Library >", did you mean one of:

Comment (penfriend) [reply cypress/e2e/orders/receive-piece-from-order.cy.js:54]: Generally Great job!!!

**Pull request merged**